terms.html

```diff
@@ -286,14 +286,18 @@
 is thus another type of verification method, even though descriptions of the process
 might not use the words "verification" and "proof."
     </p>
+  </dd>
+
+  <dt><dfn data-lt="">verification relationship</dfn></dt>
```

Comment:
this is really needed, the current spec is hard to understand without this definition.